app/routes/missions.chime/route.mdx

```diff
@@ -5,9 +5,66 @@ handle:
 
 import { feature } from '~/lib/env.server'
 
+import logo from './logo.svg'
+
 export async function loader() {
   if (!feature('CHIME')) throw new Response(null, { status: 404 })
   return null
 }
 
-# Chime
+# Canadian Hydrogen Intensity Mapping Experiment (CHIME)
+
+<img
+  src={logo}
+  align="right"
+  width="568"
+  height="420"
+  alt="Einstein Probe logo"
+  className="grid-col-6 mobile-lg:grid-col-4 tablet:grid-col-2 desktop:grid-col-3 height-auto"
+/>
+
+**Launch Date:** September 7, 2017
+
+The [Canadian Hydrogen Intensity Mapping Experiment (CHIME)](https://chime-experiment.ca/en) is a transit radio telescope
+located in Penticton BC, Canada. It consists of 4 semi cylindrical reflectors, each with 2000
```

Comment:
Is that how Canadian cities and provinces are punctuated? If it's like American cities and states, then there should be a comma between the city and the province.

Also, "semicylindrical" is a single word.

```suggestion
located in Penticton, BC, Canada. It consists of 4 semicylindrical reflectors, each with 2000
```